src/en/developer-layer-example.md

```diff
@@ -0,0 +1,153 @@
+### Introduction
+
+Vanilla is an open source themeable, pluggable, multi-lingual forum software,
+which enables communities to host a forum for discussion topics at scale.
+Powered by PHP and MySQL Vanilla is a fine example of a 3 tiered application
+
+Database (MySQL) Middleware (PHP App) Load Balancing via HTTP interface
+
+### Create the layer for the Vanilla layer
+
+From within your local repository which is designated by the environment
+variable: $JUJU_REPOSITORY
+
+    mkdir $JUJU_REPOSITORY/layers/vanilla cd $JUJU_REPOSITORY/layers/vanilla
+
+### Determine a base layer and additional relations
+
+To best leverage existing work, it is important to choose the right base layer
+for your charm. The available layers and interfaces can be found at
+[interfaces.juju.solutions](http://interfaces.juju.solutions/).
+
+For our example, Vanilla is a PHP5 application, so the layer we will use is the
+[apache-php5](https://github.com/johnsca/apache-php) layer. This is a "runtime"
+layer that provides a common basis for specific applications to be built on top
+of. Out of the box, this will give us Apache2 running on a configurable port,
+mod-php5, and support for the [http
+interface](https://code.launchpad.net/~bcsaller/charms/+source/http/+git/http)
+for running behind a proxy for high availability and scaling. On top of that,
+we'll want to use the [mysql
+interface](https://github.com/johnsca/juju-relation-mysql). We will put these in
+a `layer.yaml` file, which tells the builder what layers and interfaces to
+combine with yours to create the finished charm: includes: ['layer:apache-php',
+'interface:mysql']
+
+### Fill in your charm layer
+
+ With that decided, we're ready to start creating our charm layer. The README,
+icon, and copyright file will be the same. The `metadata.yaml` will be the same
+except that the website relation will be managed for us, so we can leave out
+the provides section:
+name: vanilla
```

Comment:
This does not render to html properly.  I would suggest using \`\`\`yaml code fence.